src/Umbraco.Web/Models/Mapping/ContentModelMapper.cs

```diff
@@ -142,9 +142,33 @@ private static void AfterMap(IContent content, ContentItemDisplay display, IData
                 TabsAndPropertiesResolver.AddListView(display, "content", dataTypeService);
             }
 
+            var currentDocumentType = contentTypeService.GetContentType(display.ContentTypeAlias);
+            var currentDocumentTypeName = currentDocumentType == null ? String.Empty : currentDocumentType.Name;
+            var docTypeValue = String.Empty;
+
+#if (DEBUG)
+            if (canAccessUser)
+            {
+                var currentUser = userService.GetUserById(UmbracoContext.Current.Security.CurrentUser.Id);

```

Comment:
A null check is required for `UmbracoContext.Current` as noted in the other usages of this singleton in this class. It is not ideal to use this singleton at all in this class but to achieve that would require some refactoring around how these mapping classes are created - in v8 this will be done with IoC and we can use the IUmbracoContextAccessor instead.

I am reviewing and modifying this PR so no need to take any actions.